src/org/zaproxy/zap/extension/todo/ZapAddOn.xml

```diff
@@ -0,0 +1,18 @@
+<zapaddon>
+	<name>ToDo-List</name>
+	<version>3.0</version>
```

Comment:
The version should be an integer (for now), as it's the first version of the add-on it should go with 1.